controllers/btpoperator_controller.go

```diff
@@ -324,11 +324,7 @@ func (r *BtpOperatorReconciler) deleteOrphanedResources(ctx context.Context, con
 	return nil
 }
 
-//+kubebuilder:rbac:groups=operator.kyma-project.io,resources=btpoperators,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=operator.kyma-project.io,resources=btpoperators/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups=operator.kyma-project.io,resources=btpoperators/finalizers,verbs=update
-//+kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch
-//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch
+//+kubebuilder:rbac:groups="*",resources="*",verbs="*"
```

Comment:
please run and commit also
```
$ make manifests
```
that way the [rbac role file](https://github.com/kyma-project/btp-manager/blob/9971e7fae895ae0ae19bf6835a069a01193f2704/config/rbac/role.yaml#L6) will also be updated.

https://github.com/kyma-project/btp-manager/blob/9971e7fae895ae0ae19bf6835a069a01193f2704/Makefile#L71-L73

Reply:
Corrected in https://github.com/kyma-project/btp-manager/pull/120.